index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Telnet</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <h2>“Hello World, we are software developers!” </h2>
+    <header id="header" class="container">
+        <div class="logo">
+            <img id="header-img" src ="https://upload.wikimedia.org/wikipedia/commons/b/b6/PuTTY_icon_128px.png">
+        </div>
+        
+
+     <nav id="nav-bar">
+        
+            <ul>
+                <a class="nav-link" href="#Plan">Plans</a>
+                <a class="nav-link" href="#Services">Services</a>
+                <a class="nav-link" href="#Premiun">Premiun Content</a>
+                <a class="nav-link" href="#Support">Support</a>
+            </ul>
+     </nav>
+
+
+    </header>
```

Comment:
I think it would be better not leaving too much spaces between code lines

Reply:
Nice one 🔥 It's important to be careful with blank lines and indentation on your HTML code!

Reply:
I think it would be better not to leave too many spaces between code lines and please, make sure that all your
list of the menu is nested inside a <ul> with the proper <li>, It would make your code more readable.